src/main/java/pl/gov/coi/pomocua/ads/dictionaries/domain/City.java

```diff
@@ -10,22 +12,21 @@
     @SequenceGenerator(name="city_gen", sequenceName="city_seq")
     private Long id;
 
-    private String voivodeship;
-    private String city;
+    @Embedded
+    @AttributeOverrides({
```

Comment:
I think that it's the default behaviour of @Embedded - when you omit that, those columns would be just `city` and `voivodeship`

Reply:
done